hbase-client/1.1.2.wso2v1/pom.xml

```diff
@@ -0,0 +1,212 @@
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.apache.hadoop.hbase.*;version="${hbase.version}";-split-package:=merge-first,
+                        </Export-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Import-Package>
+                            !org.apache.hadoop.hbase.*,
+                            org.apache.hadoop.security;version ="${hadoop.import.version}",
+                            org.apache.hadoop.security.authentication.util;version ="${hadoop.import.version}",
+                            org.apache.hadoop.security.token;version ="${hadoop.import.version}",
+                            org.apache.hadoop.net;version ="${hadoop.import.version}",
+                            org.apache.hadoop.mapreduce;version ="${hadoop.import.version}",
+                            org.apache.hadoop.io.compress.zlib;version ="${hadoop.import.version}",
+                            org.apache.hadoop.io;version ="${hadoop.import.version}",
+                            org.apache.hadoop.conf;version ="${hadoop.import.version}",
+                            org.apache.hadoop.fs;version ="${hadoop.import.version}",
+                            org.apache.hadoop.ipc;version ="${hadoop.import.version}",
+                            org.apache.hadoop.mapred;version ="${hadoop.import.version}",
+                            org.apache.hadoop.util;version ="${hadoop.import.version}",
+                            org.apache.commons.codec.binary;version ="${commons.codec.import.version}",
+                            org.apache.zookeeper.*;version="${zookeeper.import.version}",
+                            com.google.common.*;version ="${guava.import.version}",
+                            com.google.protobuf;version ="${protobuf.import.version}",
+                            org.apache.htrace.*;version="${htrace.import.version}",
+                            *;resolution:=optional
+                        </Import-Package>
+                        <Include-Resource>
+                            {maven-resources},
+                            @hbase-common-${hbase.version}.jar!/hbase-default.xml,
+                        </Include-Resource>
+                        <!-- Dynamic imports is required here because hbase-client consumes services from Hadoop SPIs -->
```

Comment:
HBase-client uses the service "org.apache.hadoop.fs.FileSystem" offered by the hadoop client to  dynamically identify and load the actual implementation class corresponding to the underlying file system (e.g. org.apache.hadoop.fs.LocalFileSystem, org.apache.hadoop.hdfs.DistributedFileSystem etc.). In the previous hadoop-client bundle, we had not included the META-INF/services directory, which caused issues in the HBase client, since the service could not be located. In the current PR, I've included the services and added dynamic imports in the consuming bundle (hbase-client).